### PR TITLE
Kudos api documentation update

### PR DESCRIPTION
**Pull request by weerd**
#### What's this PR do?

Totally forgot to update the API documentation in PR #6649 because I accidentally deleted the PR template indicator... so this PR updates that infoz.
#### How should this be reviewed?

Does the documentation look correct?
#### Any background context you want to provide?

I am silly 😬 
#### Relevant tickets

Refs #6649
#### Checklist
- [x] Documentation added for new features/changed endpoints.

---

@DFurnes @angaither @aaronschachter

### CODE REVIEW

File: documentation/endpoints/kudos.md [hunk 1]
```diff
@@ -20,16 +21,75 @@ GET https://www.dosomething.org/api/v1/kudos
 
 ### Optional query parameters
 
+**NOTE: The following query parameters no longer work with this endpoint and will soon be deprecated or refactored!**
```
Comment (DFurnes) [inline documentation/endpoints/kudos.md:24]: "no longer work with this endpoint"? Should they be removed from the docs, or am I misunderstanding?

File: documentation/endpoints/kudos.md [hunk 2]
```diff
@@ -20,16 +21,75 @@ GET https://www.dosomething.org/api/v1/kudos
...
+
 - **reportbackitems** _(string|csv)_
```
Comment (weerd) [inline documentation/endpoints/kudos.md:26]: A little concerned about this one... @aaronschachter do you recall if we use this anywhere? I wanna say it was added when Kudos was originally created for the mobile app, but then was scrapped, so not 100% sure.

File: documentation/endpoints/kudos.md (hunk 1 above)
Comment (weerd) [inline documentation/endpoints/kudos.md:24]: Well, they're still passable... and results kinda vary. Leaving them in so if _used_ it's understood that shit might be a little weird in the response. I tried it and you get a response... but it's slightly deceiving in what it replies. Once I remove those URL params, I'll fully remove them from the docs.

Comment (DFurnes) [inline documentation/endpoints/kudos.md:24]: Haha, dangerous! Okay! 😁

Comment (weerd) [inline documentation/endpoints/kudos.md:24]: 💥

**Comment by DFurnes** [comment]
📚 👍

**Pull request merged**